pkg/platform/runtime/store/store.go

```diff
@@ -180,6 +179,34 @@ func (s *Store) StoreRecipe(recipe *inventory_models.Recipe) error {
 	return nil
 }
 
+func (s *Store) CommitID() (string, error) {
+	contents, err := fileutils.ReadFile(s.markerFile())
+	if err != nil {
+		return "", locale.WrapError(err, "err_deploy_uninstall_marker", "Unable to read marker file at [ACTIONABLE]{{.V0}}[RESET]. Deployment may be corrupted.", s.markerFile())
+	}
+
+	lines := strings.Split(string(contents), "\n")
+	if len(lines) < 1 {
+		return "", locale.NewError("err_deploy_uninstall_marker", "Marker file is incomplete, cannot determine commitID")
```

Comment:
This isn't really an error. We should 100% expect this to happen as older installations wouldn't have this.